arctic/tickstore/tickstore.py

```diff
@@ -238,7 +246,7 @@ def _read_preference(self, allow_secondary):
         return ReadPreference.NEAREST if allow_secondary else ReadPreference.PRIMARY
 
     def read(self, symbol, date_range=None, columns=None, include_images=False, allow_secondary=None,
-             _target_tick_count=0):
+             _target_tick_count=0, metadata=False):
```

Comment:
This could well be in a `read_metadata` method (like in `VersionStore`) rather than overloading the return types on `read`.

Reply:
yeah that makes sense. I'll change it to its own method